.github/docker/centreon-web/jammy/configuration/admin.json

```diff
@@ -0,0 +1,7 @@
+{
+  "admin_password": "$2y$10$N3j.O9U4xEFfJMfB/g3qo.XOR/Ea7FDV418dIOyavB3EXMPshCCl6",
```

Comment:
you could also use a gh variable to avoid false hardcoded password notification ;)

Reply:
this is `Centreon!2021` default container password hashed with BCRYPT so I think this should not be in github vars / secrets